lightning/0301-update-codecov-version.patch

```diff
@@ -0,0 +1,32 @@
+From deff41925513920817a3e53c7b6aa748ef467e10 Mon Sep 17 00:00:00 2001
+From: Deepali Chourasia <deepch23@in.ibm.com>
+Date: Thu, 20 Apr 2023 08:30:36 +0000
+Subject: [PATCH] update codecov version
+
+---
+ requirements/pytorch/test.txt | 4 +---
+ 1 file changed, 1 insertion(+), 3 deletions(-)
+
+diff --git a/requirements/pytorch/test.txt b/requirements/pytorch/test.txt
+index a6e540db4..f362d2b64 100644
+--- a/requirements/pytorch/test.txt
++++ b/requirements/pytorch/test.txt
+@@ -1,5 +1,5 @@
+ coverage==6.5.0
+-codecov==2.1.12
++codecov==2.1.13
+ pytest==7.2.0
+ pytest-cov==4.0.0
+ pytest-forked==1.4.0
+@@ -8,8 +8,6 @@ pytest-rerunfailures==10.3
+ # needed in tests
+ cloudpickle>=1.3, <2.3.0
+ scikit-learn>0.22.1, <1.2.1
+-onnx<1.14.0
```

Comment:
Why these are removed? Our versions of onnx and onnxruntime fit into this requirement constraint.

Reply:
there was a failure when we have these being installed for tests. So I have removed these as we are not running tests that need them.

Reply:
okay